src/org/traccar/Protocol.java

```diff
@@ -16,4 +21,9 @@
 
     void initTrackerServers(List<TrackerServer> serverList);
 
+    Collection<String> getSupportedSmsCommands();
+
+    void sendSmsCommand(String phone, Command command) throws RecoverablePduException, UnrecoverablePduException,
```

Comment:
I think it's better to declare generic `Exception` here as well. Otherwise too many things to catch. Maybe we should create an issue in cloudhopper repository to suggest a base cloudhopper exception for all of their exceptions.